scenario-compiler/src/main/scala/pl/touk/nussknacker/engine/ModelData.scala

```diff
@@ -228,9 +238,8 @@ trait ModelData extends BaseModelData with AutoCloseable {
   // See parameters of implementing functions
   def extractModelDefinitionFun: ExtractDefinitionFun
 
-  final def modelDefinition: ModelDefinition = withThisAsContextClassLoader {
+  final def modelDefinition: ModelDefinition =
```

Comment:
This `withThisAsContextClassLoader` was some mistake, because `modelDefinitionWithClasses` is already created with  `withThisAsContextClassLoader`